spec/support_port_scanner_spec.rb

```diff
@@ -64,9 +64,11 @@ def connect_mock(_, addr)
       after = Time.now.utc
 
       elapsed = after - before
-      expected_max_time = ::Unobtainium::Support::PortScanner::MAX_RETRIES \
+      expected_min_time = ::Unobtainium::Support::PortScanner::MAX_RETRIES \
                           * ::Unobtainium::Support::PortScanner::RETRY_DELAY
-      expect(elapsed).to be <= expected_max_time
+      expected_max_time = ::Unobtainium::Support::PortScanner::MAX_RETRIES \
+                          * ::Unobtainium::Support::PortScanner::RETRY_DELAY * 2
+      expect(elapsed).to be_between expected_min_time, expected_max_time
```

Comment:
I'd prefer having brackets set as this is easier to parse (with at least my eyes):

`expect(elapsed).to be_between(expected_min_time, expected_max_time)`